src/states/node.rs

```diff
@@ -516,8 +516,8 @@ impl Node {
                 if let Ok(public_id) = verify_signed_public_id(serialised_public_id, signature) {
                     self.handle_client_identify(public_id, peer_id, client_restriction).to_evented()
                 } else {
-                    warn!("{:?} Signature check failed in ClientIdentify - Dropping connection \
-                           {:?}",
+                    warn!("{:?} Signature check failed in ClientIdentify, so dropping connection \
+                           {:?}.",
```

Comment:
I'm not too bothered either way.  I was going for consistency, but also trying to make them closer to proper sentences since they're intended to be user-facing.